shell/browser/api/electron_api_browser_window.cc

```diff
@@ -44,8 +44,8 @@ BrowserWindow::BrowserWindow(gin::Arguments* args,
 
   // Copy the transparent setting to webContents
   v8::Local<v8::Value> transparent;
-  if (options.Get("transparent", &transparent))
-    web_preferences.Set("transparent", transparent);
+  if (options.Get(options::kTransparent, &transparent))
+    web_preferences.Set(options::kTransparent, transparent);
```

Comment:
I don't like that this is copied as a string into the dictionary. Can we find some other way to propagate this information?

The way it is, this would allow e.g. specifying BrowserWindowConstructorOptions like `{ webPreferences: { transparent: true } }`, with unknown behavior (webcontents transparent but browserwindow not).

Reply:
Looks like this is a problem for `backgroundColor` as well. `SetHidden`/`GetHidden` could be used to prevent it. Maybe  there's a better approach tho.